### ISSUE TITLE
Why does RFG generate a 48x48 version inside the .ico file?

### ISSUE THREAD

**Issue by tigt** [RealFaviconGenerator/realfavicongenerator#155]
From [the FAQ](http://realfavicongenerator.net/faq#why_ico_so_big):

> The generated favicon.ico contains the icon in 16x16, 32x32 <mark>and 48x48</mark>. You may wonder what happens in extreme cases, like a 300% zoom in Windows 8.1 in HiDPI. It appears that 32x32 is enough.

And later:

> He also tried high resolution on MacOS with Chrome, Opera 15+, Opera 12.16, Safari, OmniWeb and iCab: <mark>none of them used a resolution higher than 32x32</mark>.

If the 48x48 version isn't used by anything, why is it included?

**Comment by phbernard** [comment]
You're right, explanations are not clear. In addition to the FAQ, I would like to mention that the <a href="http://realfavicongenerator.net/favicon_compatibility">compatibility test results</a> reveal no usage for this icon. So why including it?

Because Microsoft recommends it. As I only have Windows 8 to do testing, I can't rely on my observations alone to pretend the 48x48 icon is useless. So I keep it. This is an opinionated approach.

That may change with Windows 10: since this version will be mostly free, it should be widely adopted. And if this version does not use the 48x48 icon at all, maybe I'll remove it. So let's talk about this again in a year :)

**Comment by tigt** [comment]
If you like, I could test on Windows 7; I doubt it's much different from
Vista, and XP probably isn't supported.

**Comment by phbernard** [comment]
Yep, I'm interested! It's all here: http://realfavicongenerator.net/favicon_compatibility_test

**Comment by tigt** [comment]
Found it! Steps to reproduce:
- IE 11 on Windows 7
- Right click on page > Create Shortcut
- Depending on your Desktop settings, if you selected View > Large Icons,
  the 48x48 .ico is used

However, the 48x48 doesn't seem to fit in the space well at all; it's
96x96! It adds a bunch of margin around it to compensate.

When used Medium Icons in Desktop View menu, the available space is 48x48,
but it uses the 32x32 icon on a graphic of a folded page instead.

This is weirder than I could have guessed

**Comment by tigt** [comment]
Wait, even worse; it uses the 16x16 with the page graphic, not the 32x32

**Comment by tigt** [comment]
Messing with it further, it looks like there's problems with caching affecting the outcome, things are even harder to track down
